packages/ibm-products/src/components/DataSpreadsheet/DataSpreadsheet.tsx

```diff
@@ -295,11 +295,40 @@ export let DataSpreadsheet = React.forwardRef(
       [cellEditorValue, onDataUpdate]
     );
 
+    const getHeaderKeyText = useCallback((element: any) => {
```

Comment:
If this function just tries to get the text of a given node, we have a utility for that which I think would make most sense to use here, `getNodeTextContent` (`packages/ibm-products/src/global/js/utils/getNodeTextContent`).

Reply:
Hi Matthew, I took a look at this and this returns back the entire text content but in our case we have a new line like this where the header name is actually `COLUMN_1` and below it is sort of like a subheader we provide to it but in the context of wanting to know the header text it does not matter to us. 

![Screenshot 2024-08-02 at 1 32 00 PM](https://github.com/user-attachments/assets/f0585eed-da44-4530-a0f2-ae3dd94cde5c)